src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,56 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
 public class WorkWithFile {
+    private static final String COMMA = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] textFromFile = readFromFile(fromFileName);
+        String report = generateReport(textFromFile);
+        writeToFile(report, toFileName);
+    }
+
+    private String[] readFromFile(String fromFileName) {
+        try {
+            return Files.readString(Path.of(fromFileName)).split("\r?\n");
```

Comment:
Files not supported closing resources automatically
It's better to use BufferedReader and try with resources